cinnabar/plotting.py

```diff
@@ -342,6 +373,9 @@ def plot_DGs(
     plotly: bool = False,
     centralizing: bool = True,
     shift: float = 0.0,
+    bootstrap_x_uncertainty: bool = False,
+    bootstrap_y_uncertainty: bool=False,
```

Comment:
Just minor style fix.
```suggestion
    bootstrap_y_uncertainty: bool = False,
```

Reply:
Good catch, thanks!